(DOCSP-32426): Swift: Query MongoDB Test Fixes and Remove PBS from Examples (#2970)
mongodb/docs-realm: source/sdk/swift/app-services/mongodb-remote-access.txt

> describe coffee drinks for sale in a group of stores <ios-mongodb-example-dataset>`, 

Reviewer: While you're cleaning this page up, maybe reconsider having the ref cover this much text. Hyperlink text is considered non-breaking in HTML, which can introduce weird layout issues when there's a lot of link text.

I see this happens in other parts of the page too.

where the document's ``name`` field contains the value "Americano", sorted 
in descending order by the ``beanRegion`` field:

.. io-code-block::

   .. input:: /examples/generated/code/start/MongoDBRemoteAccess.snippet.find-and-sort.swift
      :language: swift

   .. output::
      :language: text

>       Results: 

Reviewer: The results bit feels a little odd because the test creates two Americano objects, but we don't indicate that anywhere in this code snippet. Readers have no idea how many Americano coffees there are.

This make the two lines of the same thing in the results like a typo. I think we should add a comment in the code to indicate there should be two coffees or add a third line to results to make it feel more intentional.

The ``.watch()`` method can take a :swift-sdk:`ChangeEventDelegate 
<Protocols/ChangeEventDelegate.html>` to subscribe to changes on a stream.
For example, with this ``ChangeEventDelegate``:

.. literalinclude:: /examples/generated/code/start/MongoDBRemoteAccess.snippet.change-event-delegate.swift
   :language: swift
   :copyable: false

This code watches for changes to documents in the ``CoffeeDrinks`` 

> collection:

Reviewer: In the "Watch for Changes in a Collection to Specific IDs" section, this introductory bit says something like this:
> The code below uses this delegate to watch for changes...

Should we add that here too? I was a little confused by the transition between the delegate example and the watch for changes example in this spot.

.. io-code-block::

   .. input:: /examples/generated/code/start/MongoDBRemoteAccess.snippet.watch-collection.swift
      :language: swift

>    .. output::

Reviewer: I think we can use `:visible:` to control whether the output is visible by default. I think it may be a good idea to hide larger outputs by default - letting people expand them if they're interested.

:ref:`database event <database-events>` that occurs while watching the
collection.

Consider an example that uses this :swift-sdk:`ChangeEventDelegate 
<Protocols/ChangeEventDelegate.html>`:

.. literalinclude:: /examples/generated/code/start/MongoDBRemoteAccess.snippet.change-event-delegate.swift
   :language: swift
   :copyable: false

> To use the code below to watch for changes to documents in the
> ``CoffeeDrink`` collection, but only trigger the provided callback for
> events corresponding to documents belonging to the ``storeNumber`` named
> ``42``:

Reviewer: This sentence is a lot. Can we break it down?
What do you think about this?
```suggestion
The code below watches for changes to documents in the
``CoffeeDrink`` collection. It only triggers the provided callback for
events related to documents that belong to the ``storeNumber`` named
``42``:
```

Author: Yeah, you're right, good catch, thank you! I broke this down a little more.
